e2e/testcases/helm_sync_test.go

```diff
@@ -439,7 +327,8 @@ image:
 		testpredicates.HasExactlyImage("wordpress", "bitnami/wordpress", "", "sha256:362cb642db481ebf6f14eb0244fbfb17d531a84ecfe099cd3bba6810db56694e"),
 		testpredicates.DeploymentHasEnvVar("wordpress", "WORDPRESS_USERNAME", "test-user-2"),
 		testpredicates.DeploymentHasEnvVar("wordpress", "WORDPRESS_EMAIL", "test-user@example.com"),
-		testpredicates.DeploymentHasEnvVar("wordpress", "TEST_INLINE", "inline"),
+		testpredicates.DeploymentHasEnvVar("wordpress", "TEST_1", "val1"),
```

Comment:
what happened to TEST_INLINE?

Reply:
The previous inline manifest defined `TEST_INLINE`, the common manifest declares these `extraEnvVars`. I don't think the name of the variables was that important, it's just asserting that the env vars from `values` persists.

The mutator could be updated to change the name of the env vars, but it seemed unnecessary